src/stock.rs

```diff
@@ -116,8 +120,28 @@ pub enum StockMessage {
     DeviceMessages = 68,
     #[strum(props(fallback = "Saved messages"))]
     SavedMessages = 69,
+
+    #[strum(props(fallback = "These messages in this chat \
+                              are generated locally by the Delta Chat app.\n\n\
+                              Delta Chat does not have your e-mail-address."))]
```

Comment:
suggestion: "Messages in this chat are generated locally by your Delta Chat app. Its makers use it to inform about app updates and problems during usage." 

general sidenote: Let's be careful with  using "Delta Chat" to vaguely refer to a remote developer/platform entity which is what i think the second sentence does.